kroxylicious-kms-provider-aws-kms/src/main/resources/META-INF/services/io.kroxylicious.kms.service.KmsService

```diff
@@ -0,0 +1,7 @@
+#
+# Copyright Kroxylicious Authors.
+#
+# Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+#
+
+io.kroxylicious.kms.provider.aws.kms.AwsKmsKmsService
```

Comment:
Yeah the name is funny, but it adheres to our <provider specific name> + `kms` convention!

Reply:
HMm I mentioned this else where, I'm not a fan I think we should drop the duplication.